README.md

```diff
@@ -1,39 +1,63 @@
 # testing-farm-as-github-action
 
-Testing Farm as GitHub Action is a GitHub Action for executing tests on the Testing Farm infrastructure.
+Testing Farm as GitHub Action is a GitHub Action for executing tests on the [Testing Farm Service](https://docs.testing-farm.io).
 
-The tests to run are to be described with a [`fmf` plan](https://tmt.readthedocs.io/en/latest/spec.html) by the user of this GitHub Action.
+The tests to run are to be described with a [`tmt` plan](https://tmt.readthedocs.io/en/latest/spec.html) by the user of this GitHub Action.
 Pull Request status is automatically updated after the tests are executed,
 if this option is enabled with the `update_pull_request_status` user-defined input variable.
 
-
 Before calling this GitHub Action, you must first clone your project,
 e.g., with the [Checkout V2](https://github.com/actions/checkout) GitHub Action.
 
-The Action uses ubuntu-20.04 and it is a [composite](https://docs.github.com/en/actions/creating-actions/about-custom-actions).
+The Action uses ubuntu-20.04 and it is a [composite action](https://docs.github.com/en/actions/creating-actions/about-custom-actions).
 It internally downloads needed binaries `curl` and `jq` for communicating with the Testing Farms API and parsing the responses.
  
-API key to the Testing Farm should be stored in your organization's secrets to successfully access its infrastructure.
+API key to the Testing Farm MUST be stored in your organization's secrets to successfully access its infrastructure.
+
+## Compatibility Notes
+
+⚠ Currently only testing with copr artifacts is supported.
```

Comment:
It would be nice to have a description of what artifacts mean. Or link.

Reply:
I added link, but we do not have it yet documented. I opened issue on our side: TFT-1041